Plugins/Util/Util.cpp

```diff
@@ -258,4 +262,13 @@ ArgumentStack Util::EncodeStringForURL(ArgumentStack&& args)
     return stack;
 }
 
+ArgumentStack Util::Get2DARowCount(ArgumentStack&& args)
+{
+    ArgumentStack stack;
+    const auto twodaRef = Services::Events::ExtractArgument<std::string>(args);
+    auto twoda = Globals::Rules()->m_p2DArrays->GetCached2DA(twodaRef.c_str(), true);
+    Services::Events::InsertArgument(stack, twoda->m_nNumRows);
```

Comment:
This crashes if you pass an empty string or a 2da that doesn't exist. This should fix it:

`Services::Events::InsertArgument(stack, !twoda ? 0 : twoda->m_nNumRows);`

Reply:
Sure, just please write it as `twoda ? twoda->m_nNumRows : 0`. :)